lib/request-proxy.js

```diff
@@ -70,7 +75,10 @@ proto.proxyReq = function proxyReq(dest, req, res, opts) {
             headers: headers,
             method: method,
             httpVersion: httpVersion,
-            checksum: ringpop.membership.checksum
+            checksum: ringpop.membership.checksum,
+            ringpopKey: key,
+            ringpopSource: ringpop.whoami(),
+            ringpopDest: dest
```

Comment:
dest is not needed as it's available as `hostInfo` on the remote side.

Let's not send it unless you want to do packet inspection or something.